sources/packages/backend/apps/api/src/route-controllers/student-account-applications/student-account-application.aest.controller.ts

```diff
@@ -77,11 +76,7 @@ export class StudentAccountApplicationAESTController extends BaseController {
   > {
     const accountApplications =
       await this.studentAccountApplicationsService.getPendingStudentAccountApplications();
-    return accountApplications.map((accountApplication) => ({
-      id: accountApplication.id,
-      fullName: getUserFullName(accountApplication.user),
-      submittedDate: accountApplication.submittedDate,
-    }));
+    return accountApplications;
```

Comment:
The `map` must happen for every DTO conversion to avoid changes at the service level to expose unwanted properties. Please add the `map` back.